DESCRIPTION

```diff
@@ -48,7 +48,7 @@ Language: en-GB
 Depends: R (>= 3.1.0)
 Imports:
     dust (>= 0.6.2),
```

Comment:
```suggestion
    dust (>= 0.6.5),
```

Reply:
May not be necessary?

Reply:
that's just changes to examples, so should be fine (though it will end up being installed about that version I think due to mcstate depending on it)